xmind/tests/test_sheet_element.py

```diff
@@ -0,0 +1,402 @@
+import logging
+from xmind.core.sheet import SheetElement
+from xmind.tests import base
+from unittest.mock import patch, Mock, MagicMock
+from xmind.core.const import *
+
+
+
+class TestSheetElement(base.Base):
+    """Test class for SheetElement class"""
+
+    def getLogger(self):
+        if not getattr(self, '_logger', None):
+            self._logger = logging.getLogger('sheetElement')
+        return self._logger
+
+    def setUp(self):
+        super(TestSheetElement, self).setUp()
+        self._init_method = self._init_patch_with_name('_init', 'xmind.core.sheet.WorkbookMixinElement.__init__')
+        self._add_id_attribute = self._init_patch_with_name('_add_id_attribute', 'xmind.core.sheet.SheetElement.addIdAttribute')
+
+    def test_excessive_parameters(self):
+        _get_root_topic = self._init_patch_with_name('_get_root_topic', 'xmind.core.sheet.SheetElement._get_root_topic')
+
+        _element = SheetElement()
+
+        _parameters = [
+            ('createRelationship', 3),
```

Comment:
some method is missing here.
you have 10 methods whenever class has 11 methods (without ```__init__```)

Reply:
It connected with _get_root_topic method which is placed in  __init__ of SheetElement class.

Reply:
and what? does it mean we shouldn't test it by excessive parameters?